learn-ocaml.el

```diff
@@ -291,8 +310,34 @@ To be used as a `make-process' sentinel, using args PROC and STRING."
 
 (defun learn-ocaml-client-version ()
   "Run \"learn-ocaml-client --version\"."
+  (string-trim (shell-command-to-string
+                (concat (shell-quote-argument learn-ocaml-command-name) " --version"))))
+
+(defun learn-ocaml-client-sign-in-cmd (server login password)
+  "Run learn-ocaml-client init-user with login and password as argument"
+  (shell-command-to-string
+   (concat (shell-quote-argument learn-ocaml-command-name)
+           " init-user --server=" server " " login " " password)))
+
+(defun learn-ocaml-client-sign-up-cmd (server login password nickname secret)
+  "Run learn-ocaml-client init-user with login password nickname
+and secret as argument"
+    (shell-command-to-string
+     (concat (shell-quote-argument learn-ocaml-command-name)
+             " init-user --server=" server " "
+             login " " password " " nickname " " secret)))
+
+(defun learn-ocaml-client-config-cmd ()
+  "Run \"learn-ocaml-client server-config\"."
+  (shell-command-to-string
+   (concat (shell-quote-argument learn-ocaml-command-name) " server-config")))
+
+(defun learn-ocaml-client-init-server-cmd (server)
+  "Run \"learn-ocaml-client server-config\"."
   (shell-command-to-string
-   (concat (shell-quote-argument learn-ocaml-command-name) " --version")))
+   (concat
+    (shell-quote-argument learn-ocaml-command-name) " init-server -s " server)))
+
```

Comment:
@LouisAyroles 

I'm sorry but I disagree here with the use of `shell-command-to-string`.
Indeed, what happens if the password or secret or… contains a single `"` char? or worse, contains `; rm -fr ~/` ?

Admittedly it might be OK to use `shell-command-to-string` for learn-ocaml-client-config-cmd as there's no extra user-provided input.

But for sign-in/sign-up/init-server, I'd prefer that you use `make-process` 🙂 (possibly with a small dedicated wrapper if it is easier for writing your function calls).